SwiftyTesseract/SwiftyTesseract/SwiftyTesseract.swift

```diff
@@ -117,11 +117,11 @@ public class SwiftyTesseract {
   ///   - bundle: The bundle that contains the tessdata folder - default is .main
   ///   - engineMode: The tesseract engine mode - default is .lstmOnly
   public convenience init(languages: [RecognitionLanguage],
-              bundle: Bundle = .main,
```

Comment:
Since this is a source breaking change, instead of changing the primary constructors we should deprecate them with `@available(*, deprecated, message: "migrate to init(dataSource:)")` and then make the deprecated constructors call into into the new ones.

Reply:
Good call, I will add a convenience initializers with deprecation warning